daemons/compute/client-go/main.go

```diff
@@ -43,6 +45,7 @@ func main() {
 	socket := flag.String("socket", "/var/run/nnf-dm.sock", "socket address")
 	maxWaitTime := flag.Int64("max-wait-time", 0, "maximum time to wait for status completion, in seconds.")
 	count := flag.Int("count", 1, "number of requests to create")
+	cancelRequests := flag.Bool("cancel", false, "attempt to cancel the requests after creation")
```

Comment:
I wonder if this should be made into a `cancelExpiryTime` so you can test any timeout length whenever you want. Useful for testing various use cases like time=0 (immediate cancel), time=short, time=long (after it's already completed). A negative value could mean "don't cancel"

Reply:
I had this same thought at one point but didn't act on it. I will now.